src/index.scss

```diff
@@ -222,43 +222,19 @@ $collapseHoverColor: #353535;
 }
 .glimpse-ajax-rows {
     font-size: type-scale(-1) !important;
+    overflow: hidden !important;
 }
 .glimpse-section-summary {
     .glimpse-ajax-rows {
         margin-left: 20px !important;
         width: 180px !important;
         flex-grow: 1 !important;
+        table-layout: fixed !important;
```

Comment:
Yes, otherwise the table doesn't obey `180px`. I'd like to fix the width so it stops jumping around.